recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "MIT+send-patches-upstream"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "pkg_config"
```

Comment:
You need to add pkgconf as a build requirement, because pkg-config is not always available.